### PR TITLE
[HIRFUS-110] Product Array Algorithm

### PR DESCRIPTION
**Pull request by AaronAsborn**
My solution to the Algoritms and Data Structure

### CODE REVIEW

File: submissions/aaronMubanga/algorithm/Algorithems.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+   console.log(productsName.length);
+
+   document.write( productsName.length + "<br><br>");
+
+   function evenPrices() {
+
+    for (let i = 0; i < productsName.length; i++) {
+
+        for (let j = 0; j < ProductPrice.length; j++) {
```
Comment (Mayowa2020) [inline submissions/aaronMubanga/algorithm/Algorithems.js:54]: Well done @AaronAsborn! The question states "Check the length of the product names in your products array". You are not expected to check for the length of the ProductPrice.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
@AaronAsborn   Kindliy effect the changes and revert when done.
Please feel free to tag me @Mayowa2020 on GitHub,  or @Bukky on Slack.

**Comment by AaronAsborn** [comment]
Screenshot 2022-08-12 at 13-24-20 Document

**Review by Mayowa2020 — APPROVED** [review]
Well done @AaronAsborn ! Approved...

**Comment by AaronAsborn** [comment]
Thank you.!!

**Pull request merged**